packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -444,6 +445,44 @@ describe('internal-plugin-metrics', () => {
       prepareDiagnosticMetricItem(webex, item);
       assert.deepEqual(item.eventPayload.origin.buildType, 'prod');
     });
+
+    it('sets the upgradeChannel as "gold" when the buildType is "prod"', () => {
+      const item: any = {
+        eventPayload: {
+          event: {
+            name: 'client.exit.app',
+            eventData: {
+              markAsTestEvent: false,
+              webClientDomain: 'https://web.webex.com',
+            },
+          },
+        },
+        type: ['diagnostic-event'],
+      };
+
+      prepareDiagnosticMetricItem(webex, item);
```

Comment:
Not sure what the difference is between this test case and `prepareDiagnosticMetricItem`. So far, it seems to be the same logic of checking the `prepareDiagnosticMetricItem` function work.